File: Book/RestaurantApp/app/src/main/java/com/example/restaurantapp/MainActivity.java
package com.example.restaurantapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    CardView startesCard;
    CardView mainCards;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        startesCard= findViewById(R.id.cardview_starters);
        mainCards=findViewById(R.id.mainCorses_cardView);

        startesCard.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent startersActivityIntent = new Intent(MainActivity.this, StartersActivity.class);
                startActivity(startersActivityIntent);
            }
        });

    }
}

File: CriminalIntent/app/src/main/java/com/example/criminalintent/MainActivity.kt
package com.example.criminalintent

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)



            var crimedetailFragment:CrimeDetailFragment  = CrimeDetailFragment()

            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container,
                crimedetailFragment).commit()





    }
}